src/corelibs/U2Formats/src/ClustalWAlnFormat.cpp

```diff
@@ -130,7 +130,7 @@ void ClustalWAlnFormat::load(IOAdapterReader &reader, const U2DbiRef &dbiRef, QL
             valEndPos++;
         }
         if (valEndPos != len) {    //there were numbers trimmed -> trim spaces now
-            while (valEndPos > valStartPos && buf[valEndPos] == ' ') {
+            while (valEndPos > valStartPos && (buf[valEndPos] == ' ' || buf[valEndPos] == '\t')) {
```

Comment:
Can we use TextUtils::isWhiteSpace() here, same as above?

Reply:
How valEndPos can grow and include spaces? We use !TextUtils::isWhiteSpace() on the line 129.

Reply:
> Can we use TextUtils::isWhiteSpace() here, same as above?

No. For newline this method returns `true`, it's wrong.

> How valEndPos can grow and include spaces?

Loop on line 133 is executed 0 or 1 times.